### DISCUSSION TITLE
Error in qsm reconstruction?

### DISCUSSION THREAD

**Discussion by AmyJolly** [QSMxT/QSMxT#74] [Q&A]
Hi there,

I've been working with QSMxt and seem to be running into the same error regardless of the three vendors I have data acquired from. I attach an image of a QSM map output from a 6-echo Siemens sequence. Do you have any idea what might be going wrong?
<img width="1134" alt="Screenshot 2022-10-04 at 15 45 34" src="https://user-images.githubusercontent.com/95216998/193850652-94991a13-0d79-4de6-8158-fb578d5d52c6.png">

Thanks,
Amy

**Comment by stebo85** [comment]
Dear @AmyJolly 

can you display it between -0.1 and 0.1 ?

Thank you 
Steffen

**Reply by AmyJolly** [reply]
<img width="1144" alt="Screenshot 2022-10-05 at 15 17 10" src="https://user-images.githubusercontent.com/95216998/194083197-bf07155b-bbfc-46ec-8f92-95683d6a04f8.png">

Hi Steffen,

Thanks, I am new to QSM and clearly didn't window between the correct values. I looked at -1 to 1 prior to messaging you, which upon re-review of your paper was far too large!.

Best wishes,
Amy

**Reply by stebo85** [reply]
Dear @AmyJolly,

No worries :) your data looks great.

Let us know if you need anything else.

Cheers
Steffen
